doc/howto/network_acls.md

```diff
@@ -92,22 +92,155 @@ lxc query --request GET /1.0/network-acls/my-acl
 
 ## Create an ACL
 
-Use the following command to create an ACL:
+Network ACL names must meet the following requirements:
+
+- Must be between 1 and 63 characters long.
+- Can contain only ASCII letters (a–z, A–Z), numbers (0–9), and dashes (-).
+- Cannot begin with a digit or a dash.
+- Cannot end with a dash.
+
+`````{tabs}
+````{group-tab} CLI
+
+To create an ACL, run:
+
+```bash
+lxc network acl create <ACL_name> [user.KEY=value ...] 
+```
+
+- You must provide an ACL name that meets the requirements described above.
+- You can optionally provide one or more custom `user` keys to store metadata or other information.
+
+ACLs have no rules upon creation via command line, so as a next step, {ref}`add rules <network-acls-rules>` to the ACL. You can also {ref}`edit the ACL configuration <network-acls-edit>`, or {ref}`assign the ACL to a network or NIC <network-acls-assign>`.
```

Comment:
You could mention here that an ACL with no rules defaults to blocking all traffic.

Reply:
Ah you covered this here https://github.com/canonical/lxd/pull/15274/commits/04ade3398bd8407350cf4890649d532d8da77a15